### DISCUSSION TITLE
File size is much bigger than previous versions

### DISCUSSION THREAD

**Discussion by ghost** [rsms/inter#673] [General]
Hello, I attempted to load Inter 4 via Cloudflare, but it increased the page weight from 68kb (current version of Inter served via Google Fonts) to over 350kb. Has anyone run into this issue?

 Also, do you have to load a separate file for italics?

**Comment by rsms** [comment]
Be more specific. Exactly what increased in size?

**Reply by rsms** [reply]
Keep in mind that fonts from Google Fonts are subset and loaded across several files as glyphs are needed. I.e. if your web page contains only Latin characters and a visitor types Greek into a text field, the web browser will go fetch another set of Inter fonts with Greek glyphs, causing a brief interruption or fonts briefly change. The fonts from rsms.me on the other hand are not subset; you get all glyphs in one request. So it's a trade off of complexity and behavior.

If you are hunting down every little  byte saving (which is unlikely to have a real-world effect), you can [subset Inter yourself](https://fonttools.readthedocs.io/en/latest/subset/) to perfectly match the glyphs you are using. This of course really only works well for static content.
